packages/TEMPLATE/install

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# To use if root is required
+[ "$UID" -eq 0 ] || (echo "installation requires root access"; exec sudo "$0" "$@")
+
+# Install general linux dependencies
+
+# General setup / configuration
+
+
+# Recipe for singularity images
+## Fetch the image
+singularity pull --name <image name> <singularity shub url>
+
+## Move it to the right place
+mv <image name> ~/.planutils/bin/images/
+
+
+# No need to move the "run" script -- if it exists, this will be handled for you
+# No need to install planutils dependencies
+# No need to check if already installed
```

Comment:
Maybe move the above two lines to the start of the file. That is the place where I would start thinking about this.

Reply:
Will do.